### PR TITLE
update readme with script and test updates

### PR DESCRIPTION
**Pull request by JettTech**
### Updates:
 - updates available script and testing instructions in README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,30 +15,45 @@ Vue based user interface for [elemental-chat DNA](https://github.com/holochain/e
...
+   2a. Thereafter, in the same window, run the dna for agent 1: `yarn hc:run-agent-1`
+      2b. Optionally, in a second window run the dna for agent 2: `yarn hc:run-agent-2`
+   3a. After the command from step 2 completes, serve the UI for agent 1: `yarn serve:hc-agent-1`
+      3b. Optionally, in the corresponding terminal for agent 2, serve the UI for agent 2: `yarn serve:hc-agent-2`
+### holo host (self-hosted) environment:
+   1. In one terminal window, install and configure the happs for the host agent: `yarn hc-host:gen-agent`
+   2. Thereafter, in the same window, run the dna for the host agent: `yarn hc-host:run`
+   3. After the command from step 2 completes, serve the UI for the host agent: `yarn serve:self-hosted`
+### holo hosting web environment:
+   1. In a second terminal window, spin up your conductor server.  This server can be two options: HCC or DEVELOP.
```
Comment (timotree3) [inline README.md:34]:
```suggestion
   1. In a second terminal window, spin up your Chaperone server.  This server can be two options: HCC or DEVELOP.
```

**Review by robbiecarlton — APPROVED** [review]
Looks good, once you've merged the suggestion from Timo

File: README.md [hunk 2]
```diff
@@ -15,30 +15,45 @@ Vue based user interface for [elemental-chat DNA](https://github.com/holochain/e
...
+  - holo-hosted web user: `yarn clear:hc-holo`
 
 ## How to run automated tests:
 For all tests
 - `yarn test`
 
 For unit tests
-- `yarn test:unit`
+- In a holochain environment `yarn test:unit-hc`
+- In a holo environment `yarn test:unit-holo`
```
Comment (robbiecarlton) [inline README.md:56]: these lines are ambiguous
does it mean "you have to be in a holochain/holo environment before running this command" 
or 
"running this command will put you in a holochain/holo environment"

**Review by robbiecarlton — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (JettTech) [reply README.md:56]: I mean these are the commands to run tests within the respective environments.  Will update to "**For** a <holochain/holo> environment..."

**Review by timotree3 — APPROVED** [review]

**Review by robbiecarlton — APPROVED** [review]

**Pull request merged**